applications/DEMApplication/custom_elements/spheric_particle.h

```diff
@@ -274,6 +274,14 @@ BoundedMatrix<double, 3, 3>* mSymmStressTensor;
 BoundedMatrix<double, 3, 3>* mStrainTensor;
 BoundedMatrix<double, 3, 3>* mDifferentialStrainTensor;
 
+struct StoredContactInfo {
```

Comment:
Good idea to store the contact info. Hopefully, someday, when we start to use the contact-based solving strategy, the storage and calculation could be less expensive.